tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -101,19 +153,19 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 			helper.CmdShouldPass("odo", "service", "create", "--from-file", fileName)
 
 			// now verify if the pods for the operator have started
-			pods := helper.CmdShouldPass("oc", "get", "pods", "-n", CI_OPERATOR_HUB_PROJECT)
+			pods := helper.CmdShouldPass("oc", "get", "pods", "-n", project)
```

Comment:
As we are already setting project using odo command, There is no use of explicitly mentioning `-n` flag here. So I did just a check to confirm how `odo project set` sets active projects for `oc` too.

```
$ odo project create xya
 ✓  Project 'xya' is ready for use
 ✓  New project created and now using project: xya
$ odo project list
ACTIVE     NAME
           default
           [...]
           viraj
*          xya
$ oc projects
You have access to the following projects and can switch between them with 'oc project <projectname>':

    default
    [...]
    viraj
  * xya
$ odo project set viraj
Switched to project : viraj
$ odo project list
ACTIVE     NAME
           default
           [...]
*          viraj
           xya
$ oc projects
You have access to the following projects and can switch between them with 'oc project <projectname>':

    default
    [...]
  * viraj
    xya
```

Reply:
I'm just being explicit here. Besides, in this comment https://github.com/openshift/odo/issues/3193#issuecomment-632524803 you have mentioned to provide the project name for better user experience. So can you please decide if you want to use `--project` or not?

Reply:
It seems you misunderstood my comment. NP, I have put the details in the [issue](https://github.com/openshift/odo/issues/3241) what was trying to say.

Reply:
> It seems you misunderstood my comment. NP, I have put the details in the [issue](https://github.com/openshift/odo/issues/3241) what was trying to say.

This comment thread which started from https://github.com/openshift/odo/pull/3239#discussion_r429184825 is about "not using the `-n` flag in tests", whereas that issue is about "project name not showing up in the output". How are the two things similar? How did I misunderstand what you were trying to say? What am I missing here? :thinking: 

> As part of the `--project` flag is concerned TBH it depends on how you are going to use this. Here you have already setting the active project through `odo project set...` that too in `justBeforeEach`. So it really does not make sense to pass the `--project` flag in the spec implementation.
> 
> If you want to use the `--project` flag then don't use set command. Either way implementation will work fine.

This part of your comment is about what you started out with in https://github.com/openshift/odo/pull/3239#discussion_r429184825. But the issue and this comment thread don't seem to have anything in common !